home/models.py

```diff
@@ -88,6 +88,9 @@ class Meta:
     def __str__(self):
         return f"Question #{self.id} : {self.title} ( {self.publish_date.strftime('%d/%m/%Y %H:%M')} )"
 
+    def get_answers_num(self):
+        return self.answer_set.count()
+
```

Comment:
What is answer_set? we don't have that field in Question model

Reply:
Yes, We don't have answer_set field in Question model .

The _set suffix is a reverse lookup class variable in Django.
You can read more about it in the following link https://docs.djangoproject.com/en/3.2/topics/db/examples/many_to_one/

You're welcome to ask me if you have any further questions regarding this issue .

Reply:
Cool didn't know it. Thanks!!